src/javaRosa.js

```diff
@@ -837,6 +838,34 @@ define([
             });
 
         if (options.path === 'labelItext') {
+            if (EXPERIMENTAL_UI) {
+                $input.atwho({
+                    at: "#form",
+                    data: _.chain(mug.form.getMugList())
+                           .map(function(mug) {
+                                return {
+                                    id: mug.ufid,
+                                    name: mug.form.getAbsolutePath(mug, true),
+                                    path: mug.absolutePath
+                                };
+                            })
+                            .filter(function(choice) { return choice.name; })
+                            .value(),
+                    displayTpl: '<li>${name}</li>',
+                    insertTpl: '<output value="${path}" />',
+                    limit: 100,
```

Comment:
this could be more like 10, maybe 20, don't think scrolling beyond that many options is likely